src/scripts.js

```diff
@@ -51,25 +38,25 @@ function getRandomID(array) {
 };
 
 function updateDashboard() {
-  displayWelcomeMessage();
-  displayUserInfo();
-  displayUserStepGoal();
-  displayAvgUsersStepGoal();
+  welcomeUser(currentUser);
+  updateUserInfo(currentUser);
+  domUpdates.displayUserStepGoal(currentUser);
+  domUpdates.displayAvgUsersStepGoal();
   displayDailyStats();
```

Comment:
Hey there! I would consider changing the displayDailyStats and displayWeeklyStats function to updateDailyStats and updateWeeklyStats to keep convention between our scripts file and our domUpdates file. Otherwise, looks great!